components/main/blocks/DKMembershipWidget/DKMembershipWidget.tsx

```diff
@@ -150,16 +224,37 @@ export const DKMembershipWidget: React.FC<{ config?: ConfigurationType }> = ({ c
           .json()
           .catch(() => ({ message: "Submission failed with status: " + response.status }));
         console.error("Submission error:", errorData);
+        setApiError(mergedTexts.failed_submission_message);
         setLoading(false);
       }
     } catch (error) {
       console.error("Network or other error:", error);
+      setApiError(mergedTexts.failed_submission_message);
       setLoading(false);
     }
   };
 
   return (
     <div className={styles.widgetContainer}>
+      <AnimateHeight height={apiError ? "auto" : 0} duration={300} animateOpacity>
```

Comment:
We use styled components in the donation widget mostly for historical reasons, if you notice a bit further down we actually just use classnames for styling in our blocks (where the styles are defined in the `.module.scss` file of the same name in the same folder as this component file). Confusing!

It'd be great if we could stick to one approach in this file, could you perhaps redo the styling with classnames and have the styles in the scss module file?